### PR TITLE
adds more posibrain fluffnames.

### PR DESCRIPTION
**Pull request by Militaires**
p straightforward.

:cl: Basilman
add: Added new positronic brain names
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/brain/posibrain.dm [hunk 1]
```diff
@@ -25,7 +25,7 @@ var/global/posibrain_notif_cooldown = 0
 	Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm.</b>"
 	var/new_mob_message = "<span class='notice'>The positronic brain chimes quietly.</span>"
 	var/dead_message = "<span class='deadsay'>It appears to be completely inactive. The reset light is blinking.</span>"
-	var/list/fluff_names = list("PBU","HIU","SINA","ARMA","OSI","HBL","MSO","RR","CHRI","CDB","HG","XSI","ORNG","GUN","KOR","MET","FRE","XIS","SLI","PKP","HOG","RZH","GOOF","MRPR","JJR","FIRC","INC","PHL","BGB","ANTR","MIW","WJ","JRD","CHOC","ANCL","JLLO","JNLG","KOS","TKRG","XAL","STLP","CBOS","DUNC","FXMC","DRSD")
+	var/list/fluff_names = list("PBU","HIU","SINA","ARMA","OSI","HBL","MSO","RR","CHRI","CDB","HG","XSI","ORNG","GUN","KOR","MET","FRE","XIS","SLI","PKP","HOG","RZH","GOOF","MRPR","JJR","FIRC","INC","PHL","BGB","ANTR","MIW","WJ","JRD","CHOC","ANCL","JLLO","JNLG","KOS","TKRG","XAL","STLP","CBOS","DUNC","FXMC","DRSD","XHS","NIK","BOB","KRUS","BMAN","MEME","EXAD","JOAN","JMAD")
```
Comment (ChangelingRain) [inline code/modules/mob/living/carbon/brain/posibrain.dm]: I get two? cool!

Comment (Bawhoppen) [inline code/modules/mob/living/carbon/brain/posibrain.dm]: >jmad getting one

tyranny

**Comment by Militaires** [comment]
@ChangelingRain  woops.

File: code/modules/mob/living/carbon/brain/posibrain.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/modules/mob/living/carbon/brain/posibrain.dm]: >tfw not notorious enough to get one

**Comment by KorPhaeron** [comment]
No player references sorry

**Comment by Militaires** [comment]
@KorPhaeron kor but all of the current posibrain fluffnames are playernames wat, even you are there

**Comment by Militaires** [comment]
@PKPenguin321 you already r ther doe

**Comment by PKPenguin321** [comment]
i didnt even know lol that's awesome

**Comment by Militaires** [comment]
@KorPhaeron look at the current names

KOR, HG, ORNG, MSO, CHRI, GUN, HOG, WJ, GOON, RZH

**Comment by MrStonedOne** [comment]
PURELY COINCIDENCE!!!!!

**Comment by Niknakflak** [comment]
shameful double standards

**Comment by AnonymousNow** [comment]
No player references? Really? What would you suggest instead?

`var/list/fluff_names = list("DCK","CHKN","RAVN","PDGN","EGL","SGL","FMNG","BDGR","RAT","FOX","HDGH","CAT","LION","TIGR","PNTR","CGR","LYNX","OCLT","JGUR","WOLF","MOUS","RBBT","MNAT","WHL","TUNA","DLPN","SHRK","SEAL","JLYF","MNTA","CTPS","NRWL","FROG","CRAB","SPDR","TRNT","SNK","CBRA","ADDR","PYTN","BOA","COW","PIG","BOAR","HRSE","BAT","GRLA","MNKY","CHMP","RNGT","BBN","GBBN","TRTL","TRTS","ZBRA","ANT","BEE","WASP","TRMT","WDLS","CNTP")`

Is this want you _really_ want?

**Comment by KorPhaeron** [comment]
>PURELY COINCIDENCE!!!!!

**Comment by Niknakflak** [comment]
you can have a double standard or you can just let everyone in
Don't be a hypocrite

**Comment by Militaires** [comment]
for once i agree with @Niknakflak

**Comment by MrStonedOne** [comment]
Going all out on this "if i don't get in, nobody should get in" rant is stupid and childish, I don't argue that scardy's beef jerky should get removed if I can't have my own snack food reference.

**Comment by Militaires** [comment]
@MrStonedOne  sure, who gives a shit about a couple of extra fluffnames for the posibrains anyways, it's just that kor told me that it was denied because player references arent allowed is plain untrue.

if you're gonna tell me something go all out and just say i cant have it, i'd take that for a good reason to deny the pr.

**Pull request closed without merging**